Code/Framework/AzToolsFramework/Tests/Prefab/PrefabCreateInMemoryTests.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+
+#include <Prefab/PrefabTestFixture.h>
+
+namespace UnitTest
+{
+    using PrefabCreateInMemoryTest = PrefabTestFixture;
```

Comment:
Should it be PrefabCreateTests?

Would like to have a follow-up task to put all the tests for public API under "public" folder and all undo tests under "undo" folder. 😄